bokeh/properties.py

```diff
@@ -837,7 +837,8 @@ class List(Seq):
 
     """
 
-    def __init__(self, item_type, default=[], help=None):
+    def __init__(self, item_type, default=None, help=None):
+        default = default or []
```

Comment:
Thanks for the explanation. What do you mean with optional values? As I understand it now, you want to be able to write this:

    >> class Foo(HasProps):
    >>     bar = List(default=None)
    >> foo = Foo()
    >> print(foo.bar)
    None

Isn't that strange, since then the value will not be a list?